apps/client/src/pages/QuestionsPage.js

```diff
@@ -47,6 +49,9 @@ const QuestionsPage = ({ topic, checker }) => {
 
   const onQuestionNext = (value) => {
     if (question.options) {
+      context.setData({
```

Comment:
Can you add a comment why you need to set the data on context before you set the answer to the question. Is it because of position on the stack?